src/config.rs

```diff
@@ -128,3 +128,37 @@ pub fn read_config() -> Result<UserSettings, ConfigError> {
     let config = config_builder.add_source(env_overrides()).build()?;
     Ok(UserSettings::from_config(config))
 }
+
+/// Command name and arguments specified by config.
+#[derive(Clone, Debug, Eq, Hash, PartialEq, serde::Deserialize)]
+#[serde(untagged)]
+pub enum FullCommandArgs {
+    String(String),
+    // TODO: Vec<String>
+}
+
+impl FullCommandArgs {
+    /// Returns arguments including the command name.
+    ///
+    /// The list is not empty, but each element may be an empty string.
+    pub fn split(&self) -> Vec<String> {
```

Comment:
nit: `split()` seems to describe more how it's implemented than what it is for. How about `args()` or something like that? Or maybe `argv()` if you think that better encapsulates the fact that it includes the name of the program.

Reply:
Sure, renamed to `args()`. It might be a bit confusing in this context, but Rust appears to call `argv` of `int main(argc, argv)` as `env::args()`..